opam-packages-conversion/bin/config.py

```diff
@@ -34,7 +34,8 @@
     "reactiveData",
     "deriving",
     "ocamlbuild",
-    }
+    "js_of_ocaml",
```

Comment:
Was hitting circular deps — I think we need to implement build deps to solve that.